src/views/partials/top.ejs

```diff
@@ -68,7 +68,7 @@
                         </li>
                         <li class="govuk-header__navigation-item govuk-header__navigation-item--active">
                             <a class="govuk-header__link"
-                                href="/<%= i18n.language %>/list">
+                                href="/<%= i18n.language %>/<%= t('routes.dataset') %>">
```

Comment:
We're translating the URL path as well as the labels?

Reply:
We are... its part of the welsh first thing... Remind me next week.  We did some workshops on this and theres some bits around about it which might be helpful